components/navbar.css

```diff
@@ -8,27 +8,139 @@
     flex-wrap: wrap;
     width: 100%;
     align-content: center;
-    justify-content: space-around;
+    justify-content: space-between;
     padding: 0.7rem 1rem;
+    /* position: fixed; */
+    /* top: 0; */
+    /* background-color: var(--white-color); */
```

Comment:
Please try to remove all the unnecessary  code and commented code this makes it more readable

Reply:
Yes I will remove this. Thanks for reviewing